lib/drivers/medtronic600/NGPHistoryParser.js

```diff
@@ -979,6 +988,18 @@ class NGPHistoryParser {
     return matchingReadings;
   }
 
+  findSmbgForCalibration(calibrationEvent) {
+    const calibrationIndex = _.findIndex(this.events, event => event === calibrationEvent);
+
+    return _.findLast(
+      this.events, item => (item.eventType === NGPHistoryEvent.EVENT_TYPE.BG_READING ||
+        (item.eventType === NGPHistoryEvent.EVENT_TYPE.CLOSED_LOOP_BG_READING &&
+          item.isFirstReading())) &&
+        item.bgValue === calibrationEvent.bgValue
+      , calibrationIndex,
```

Comment:
I know this is super nit-picky,  but isn't that comma in a strange place?

Reply:
Yep, it is. Thanks `eslint-config-airbnb 16.1.0`!